client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -78,9 +78,9 @@ class ListItem extends PureComponent {
 
     this.handleItemLock();
 
-    return updateListItem(itemName, listId, _id, userData, data).finally(() => {
-      this.setState({ disableToggleButton: false }, this.handleItemUnlock);
-    });
+    return updateListItem(itemName, listId, _id, userData, data).finally(() =>
+      this.setState({ disableToggleButton: false }, this.handleItemUnlock)
```

Comment:
setState in finally may cause error/warning: "can't call setState on unmounting components (...)", what if updated item is unmounting? Or we don't care about this warning any more?

Reply:
I've been testing and I didn't have this warning any time